database/database.go

```diff
@@ -65,22 +65,22 @@ func (db *Database) RecordFingerprintImportedIntoGnuPG(newFingerprint fpr.Finger
 // GetFingerprintsImportedIntoGnuPG returns a slice of fingerprints that have
 // been imported into GnuPG
 func (db *Database) GetFingerprintsImportedIntoGnuPG() (fingerprints []fpr.Fingerprint, err error) {
-	databaseMessage, err := db.getMessage()
+	message, err := db.readMessage()
 	if os.IsNotExist(err) {
 		return []fpr.Fingerprint{}, nil
 	}
 	if err != nil {
 		return nil, err
 	}
 
-	for _, v := range databaseMessage.KeysImportedIntoGnuPG {
+	for _, v := range message.KeysImportedIntoGnuPG {
 		fingerprints = append(fingerprints, v.Fingerprint)
 	}
 
 	return deduplicate(fingerprints), nil
 }
 
-func (db *Database) getMessage() (message *Message, err error) {
+func (db *Database) readMessage() (message *Message, err error) {
```

Comment:
I'm gonna suggest we go really explicit and call these
* `readMessage` -> `loadFromFile`
* `writeMessage` -> `saveToFile`